### PR TITLE
Feature/environment texture

### PR DESCRIPTION
**Pull request by angrysnails**
Added environment textures

### CODE REVIEW

File: Assets/Art/2D/Textures/Terrain/T_GrassNormalMap.png
Comment (TryFox) [inline Assets/Art/2D/Textures/Terrain/T_GrassNormalMap.png:1]: Normal map has a _N suffix that is missing here

File: Assets/Art/2D/Textures/Terrain/T_SnowNormalMap.png
Comment (TryFox) [inline Assets/Art/2D/Textures/Terrain/T_SnowNormalMap.png:1]: Normal map has a _N suffix that is missing here

**Review by TryFox — CHANGES_REQUESTED** [review]
The base color has a _D suffix, please add that to all the base color textures

File: Assets/Art/2D/Textures/Terrain/T_GrassNormalMap_N.png
Comment (TryFox) [inline Assets/Art/2D/Textures/Terrain/T_GrassNormalMap_N.png:1]: dont have to add normal map to the name

File: Assets/Art/2D/Textures/Terrain/T_GrassNormalMap_N.png.meta
Comment (TryFox) [inline Assets/Art/2D/Textures/Terrain/T_GrassNormalMap_N.png.meta:1]: dont have to add normal map to the name

File: Assets/Art/2D/Textures/Terrain/T_SnowNormalMap_N.png
Comment (TryFox) [inline Assets/Art/2D/Textures/Terrain/T_SnowNormalMap_N.png:1]: dont have to add normal map to the name

File: Assets/Art/2D/Textures/Terrain/T_SnowNormalMap_N.png.meta
Comment (TryFox) [inline Assets/Art/2D/Textures/Terrain/T_SnowNormalMap_N.png.meta:1]: dont have to add normal map to the name

**Review by TryFox — CHANGES_REQUESTED** [review]
Please change naming of Normal map texture

**Review by TryFox — APPROVED** [review]

File: Assets/Terrain/Settings/Pallets/PL_RailGras.terrainlayer
Comment (lucasvdiepen) [inline Assets/Terrain/Settings/Pallets/PL_RailGras.terrainlayer:1]: Change name `Gras` to `Grass`.

File: Assets/Art/2D/Textures/Terrain/T_RailGras_D.png
Comment (lucasvdiepen) [inline Assets/Art/2D/Textures/Terrain/T_RailGras_D.png:1]: Change name `Gras` to `Grass`.

File: Assets/Terrain/Settings/Pallets/PL_SnowGras.terrainlayer
Comment (lucasvdiepen) [inline Assets/Terrain/Settings/Pallets/PL_SnowGras.terrainlayer:1]: Change name `Gras` to `Grass`.

File: Assets/Terrain/Settings/Pallets/TP_TerrainPallet.asset
Comment (lucasvdiepen) [inline Assets/Terrain/Settings/Pallets/TP_TerrainPallet.asset:1]: `Pallette Layers` has a missing reference.

**Review by lucasvdiepen — CHANGES_REQUESTED** [review]

**Comment by lucasvdiepen** [comment]
Some missing or dangling .meta found. Fix commits are needed.

| Status | File |
|:--|:--|
| Not removed | `Assets/Art/2D/UI/Map.meta` |

**Review by lucasvdiepen — APPROVED** [review]

**Pull request merged**